tests/cli/test_rasa_data.py

```diff
@@ -60,15 +60,52 @@ def test_data_convert_help(run: Callable[..., RunResult]):
 def test_data_validate_help(run: Callable[..., RunResult]):
     output = run("data", "validate", "--help")
 
-    help_text = """usage: rasa data validate [-h] [-v] [-vv] [--quiet] [--fail-on-warnings]
-                          [-d DOMAIN] [--data DATA]"""
+    help_text = """usage: rasa data validate [-h] [-v] [-vv] [--quiet]
+                          [--max-history MAX_HISTORY] [--fail-on-warnings]"""
 
     lines = help_text.split("\n")
 
     for i, line in enumerate(lines):
         assert output.outlines[i] == line
 
 
+def _text_is_part_of_output_error(text: Text, output: RunResult) -> bool:
+    found_info_string = False
+    for line in output.errlines:
+        if text in line:
+            found_info_string = True
+    return found_info_string
+
+
+def test_data_validate_without_max_history(
```

Comment:
When you move the other `max_history` warning to `_validate_story_structure` you can these tests a bit leaner / more elegant:

```python
validator = Mock()
args = Mock()
args.max_history = -1

_validate_story_structure(validator, args)

assert warning was logged
```

Reply:
you could even use pytest.mark.parametrize for all three tests

Reply:
See my comment above...

Reply:
Sorry, GitHub unfortunately doesn't show it to me. Can you give me a link?

Reply:
Here: https://github.com/RasaHQ/rasa/pull/4968#discussion_r372293074 But it's no longer relevant, since one can now call `data test stories` without `--max-history`.